packages/chevrotain/test/full_flow/ecma_quirks/ecma_quirks.ts

```diff
@@ -85,9 +89,102 @@ function deferredInitTokens() {
 
 const ErrorToken = createToken({ name: "ErrorToken" })
 
+class EcmaScriptQuirksLookaheadStrategy implements ILookaheadStrategy {
```

Comment:
Were any meaningful changes needed here?
Or is this just wrapping the existing code with the new APIs?

Reply:
Yeah, it was necessary: The old way of overriding the (undocumented) lookahead methods doesn't work anymore, as they (`lookAheadBuilderForAlternatives` and `lookAheadBuilderForOptional`) were removed completely and are now part of the `ILookaheadStrategy` API.

Also, it serves as a means to provide a second implementation of the `ILookaheadStrategy` interface. A regression test in case something breaks internally and the API isn't called anymore or something similar.